src/Loader.php

```diff
@@ -211,6 +211,7 @@ public static function add_blocks(): void
         new Blocks\TableOfContents();//NOSONAR
         new Blocks\TakeActionBoxout();//NOSONAR
         new Blocks\Timeline();//NOSONAR
+        new Blocks\Covers();//NOSONAR
```

Comment:
This is super nicky picky, but it is not alphabetically ordered 😆

Reply:
Not nit picky, we should keep it ordered! Updated 👍